src/util/helpers.tsx

```diff
@@ -301,3 +302,21 @@ export const getFileExtension = (filename: string): string => {
 
 export const delay = (ms: number): Promise<void> =>
   new Promise((resolve) => setTimeout(resolve, ms));
+
+export const returnAllDevices = (
+  instance: LxdInstance,
+): [string, LxdDeviceValue][] => {
+  const instanceDevices = Object.entries(instance.expanded_devices ?? {});
+
+  return instanceDevices;
+};
```

Comment:
```suggestion
export const getInstanceDevices = (
  instance: LxdInstance,
): [string, LxdDeviceValue][] => {
  return Object.entries(instance.expanded_devices ?? {});
};
```
Better naming and a bit simpler?